.github/workflows/validate.yml

```diff
@@ -12,6 +12,10 @@ on:
       - 'alpha'
       - '!all-contributors/**'
   pull_request: {}
+permissions:
+  actions: write #  to cancel/stop running workflows (styfle/cancel-workflow-action)
```

Comment:
Could you include a summary of the permissions before and after for each change?

Reply:
Before the change for all jobs (as seen here https://github.com/testing-library/react-testing-library/actions/runs/3633193928/jobs/6129924592):
```
GITHUB_TOKEN Permissions
  Actions: write
  Checks: write
  Contents: write
  Deployments: write
  Discussions: write
  Issues: write
  Metadata: read
  Packages: write
  Pages: write
  PullRequests: write
  RepositoryProjects: write
  SecurityEvents: write
  Statuses: write
```
After the change
For the release job (not mention permissions are `none`):
```
GITHUB_TOKEN Permissions
  Actions: write
  Contents: write
  Metadata: read
```
For all others:
```
GITHUB_TOKEN Permissions
  Actions: write
  Contents: read
  Metadata: read
```